app/src/main/java/cm/aptoide/pt/social/commentslist/PostCommentsView.java

```diff
@@ -0,0 +1,39 @@
+package cm.aptoide.pt.social.commentslist;
+
+import cm.aptoide.pt.dataprovider.model.v7.Comment;
+import cm.aptoide.pt.presenter.View;
+import java.util.List;
+import rx.Observable;
+
+/**
+ * Created by jdandrade on 28/09/2017.
+ */
+
+public interface PostCommentsView extends View {
+
+  Observable<Object> reachesBottom();
```

Comment:
Why object and not Void?

Reply:
wouldn't I need to map the emission to null instead of the object cast? What are the advantages?